deepfence_worker/ingesters/common.go

```diff
@@ -88,7 +88,16 @@ func CommitFuncStatus[Status any](ts utils.Neo4jScanType) func(ns string, data [
 			SET cr.` + ScanCountField[ts] + `=count, cr.` + ScanStatusField[ts] + `=n.status, cr.` + LatestScanIdField[ts] + `=n.node_id`
 		}
 
-		if _, err = tx.Run(query, map[string]interface{}{"batch": statusesToMaps(data)}); err != nil {
+		maps := statusesToMaps(data)
+		for idx := 0; idx < 3; idx++ {
+			if _, err = tx.Run(query, map[string]interface{}{"batch": maps}); err != nil {
+				log.Error().Msgf("Error occured while updating scan status: %+v, Retrying...", err)
+				time.Sleep(1 * time.Second)
+				//return err
+			}
+		}
```

Comment:
This looks like a retry loop:
- rename `idx` into `retries`
- add a `break` in the `else` branch

Reply:
Actually none of this is required
If an ingestion has an error, the upper function will take care of retry

Reply:
Done.